core/src/cgcloud/core/apache.py

```diff
@@ -47,15 +52,14 @@ def __install_apache_package( self, path ):
                 else:
                     log.warn( "Could not download or extract the package, retrying ..." )
 
-    # FIMXE: this might have more general utility
-
-    def __apache_official_mirror_url( self, path ):
-        mirrors = run( "curl -fs 'http://www.apache.org/dyn/closer.cgi?path=%s&asjson=1'" % path )
+    def _apache_official_mirror_url( self, remote_path ):
+        url = 'http://www.apache.org/dyn/closer.cgi?path=%s&asjson=1' % remote_path
+        mirrors = run( "curl -fs '%s'" % url )
         mirrors = json.loads( mirrors )
         mirror = mirrors[ 'preferred' ]
-        url = mirror + path
+        url = mirror + remote_path
         return url
 
-    def __apache_s3_mirror_url( self, path ):
-        file_name = os.path.basename( path )
+    def _apache_s3_mirror_url( self, remote_path ):
```

Comment:
In spite of my stupidity RE: protected vs. private in Python, I do think that this method should be private.